src/builder.ts

```diff
@@ -29,13 +28,23 @@ export class Builder {
     }
   }
 
-  private removeDistDir() {
-    const distDir = path.resolve(this.projectDir, 'dist');
+  private getBuildConfig() {
+    const packageJson = require(path.resolve(this.projectDir, 'package.json')); // eslint-disable-line @typescript-eslint/no-var-requires
+    const leanixReportingCli = packageJson.leanixReportingCli || {};
+
+    return {
+      distPath: leanixReportingCli.distPath ?? './dist',
+      buildCommand: leanixReportingCli.buildCommand ?? 'node_modules/.bin/webpack'
+    }
+  }
+
+  private removeDistDir(distPath: string) {
+    const distDir = path.resolve(this.projectDir, distPath);
     return rimrafAsync(distDir);
   }
 
-  private buildWithWebpack() {
-    const webpackCmd = path.resolve(this.projectDir, 'node_modules/.bin/webpack');
+  private doBuild(buildCommand: string) {
+    const webpackCmd = path.resolve(this.projectDir, buildCommand);
```

Comment:
At this point in time I thought it was the path to the executable for the build command. Later I figured that the project directory is actually the current working directory, so that the `path.resolve()` is not needed. I also figured out that the build command can have additional parameters, which would probably break the `path.resolve()`... Anyway, the final implementation just takes the `buildCommand` and executes it.